test/utils/metrics/metrics.go

```diff
@@ -131,6 +131,12 @@ func CheckClockRoleAndOffset(ptpConfig *ptpv1.PtpConfig, label, nodeName *string
 	if nodeName == nil {
 		var name string
 		name, err = getNode(*label)
+		if err != nil || name == "" || label == nil || *label != utils.PtpClockUnderTestNodeLabel {
+			fmt.Printf(`error getting node name for label %s
```

Comment:
log instead of fmt

Reply:
This is the last log of the output, using fmt for better formatting